app/src/main/java/ch/epfl/polybazaar/FillListingActivity.java

```diff
@@ -150,12 +156,12 @@ else if (requestCode == RESULT_LOAD_IMAGE){
             }
 
             stringImage = convertBitmapToString(bitmap);
-            Bitmap resizedBitmap = resizeBitmap(bitmap, 0.5f, 0.5f);
-            stringThumbnail = convertBitmapToStringWithQuality(resizedBitmap, 10);
+            //Bitmap resizedBitmap = resizeBitmap(bitmap, 0.5f, 0.5f);
```

Comment:
Do you really need to keep them in comment, cant you remove them?

Reply:
I forgot to remove them!